modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.teaser.yml

```diff
@@ -0,0 +1,91 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.teaser
+    - field.field.node.oe_consultation.oe_consultation_additional_info
+    - field.field.node.oe_consultation.oe_consultation_aim
+    - field.field.node.oe_consultation.oe_consultation_closed_text
+    - field.field.node.oe_consultation.oe_consultation_contacts
+    - field.field.node.oe_consultation.oe_consultation_deadline
+    - field.field.node.oe_consultation.oe_consultation_guidelines
+    - field.field.node.oe_consultation.oe_consultation_legal_info
+    - field.field.node.oe_consultation.oe_consultation_opening_date
+    - field.field.node.oe_consultation.oe_consultation_outcome
+    - field.field.node.oe_consultation.oe_consultation_outcome_files
+    - field.field.node.oe_consultation.oe_consultation_response_button
+    - field.field.node.oe_consultation.oe_consultation_target_audience
+    - field.field.node.oe_consultation.oe_departments
+    - field.field.node.oe_consultation.oe_subject
+    - field.field.node.oe_consultation.oe_summary
+    - field.field.node.oe_consultation.oe_teaser
+    - node.type.oe_consultation
+  module:
+    - datetime
+    - field_group
+    - user
+third_party_settings:
+  field_group:
+    group_details:
+      children:
+        - oe_consultation_opening_date
+        - oe_consultation_deadline
+      parent_name: ''
+      weight: 1
+      format_type: oe_theme_helper_field_list_pattern
+      region: content
+      format_settings:
+        variant: horizontal
+      label: Details
+id: node.oe_consultation.teaser
+targetEntityType: node
+bundle: oe_consultation
+mode: teaser
+content:
+  extra_field_oe_consultation_label_status:
+    weight: 0
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  oe_consultation_deadline:
+    weight: 5
+    label: above
```

Comment:
Make sure the labels are hidden, just like on other CTs.

Reply:
Could you clarify or show me an example?
In similar config for 'Call for tenders' label also 'above'